src/components/atoms/BenefitsCounter.js

```diff
@@ -6,11 +6,11 @@ import PropTypes from "prop-types";
  */
 export function BenefitsCounter(props) {
   return (
-    <div className="text-center m-auto">
-      <div className="-mb-8" style={{ fontSize: "100px" }}>
+    <div className="text-center m-auto mr-0">
```

Comment:
minor but positioning is better left to parent element.

suggestion: Create className prop to pass in css classes so that element can be positioned via parent. So in the home page you would pass in m-auto mr-0